docs/tutorial_notebooks/tutorial5/Inception_ResNet_DenseNet.ipynb

```diff
@@ -1255,7 +1255,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The training of the model is handled by PyTorch Lightning, and we just have to define the command to start. Note that we train for almost 200 epochs, which takes about an hour on Lisa's default GPUs (GTX1080Ti). We would recommend using the saved models and train your own model if you are interested. "
+    "The training of the model is handled by PyTorch Lightning, and we just have to define the command to start. Note that we train for almost 200 epochs, which takes about an hour on Snellius's default GPUs (NVIDIA A100). We would recommend using the saved models and train your own model if you are interested. "
```

Comment:
A100s will be for sure faster than the old 1080Ti's :)
You can update the text to 'which takes less than an hour'